data/mods/Aftershock/items/weapons.json

```diff
@@ -226,5 +226,23 @@
     "volume": "1750 ml",
     "bashing": 22,
     "price": 20000
+  },
+  {
+    "id": "ceramic_knife",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": "bodyguard knife",
```

Comment:
Can you add a MA extension .json for this? There's an example in Magiclysm and Mythological weapons on how to do it.

Reply:
Do I need to add that in a separate PR?  I'm not 100% what you are asking but I'll try and figure out.  Did I use an ID or something that is in another place?